detection-rules/impersonation_domain_replyto_freemail_lookalike_financial_request.yml

```
@@ -0,0 +1,30 @@
+name: "Domain Impersonation Freemail ReplyTo_Local Lookalike with Financial Request"
```

```suggestion
name: "Domain Impersonation: Freemail ReplyTo_Local Lookalike with Financial Request"
```
might make this rule name more digestible